server/protoboards.go

```diff
@@ -0,0 +1,76 @@
+package server
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/bouk/httprouter"
+	"github.com/influxdata/chronograf"
+)
+
+type protoboardLinks struct {
+	Self string `json:"self"`
+}
+
+type protoboardResponse struct {
+	chronograf.Protoboard
+	Links protoboardLinks `json:"links"`
+}
+
+func newProtoboardResponse(protoboard chronograf.Protoboard) protoboardResponse {
+	httpAPIProtoboards := "/chronograf/v1/protoboards"
+	selfLink := fmt.Sprintf("%s/%s", httpAPIProtoboards, protoboard.ID)
+
+	return protoboardResponse{
+		Protoboard: protoboard,
+		Links: protoboardLinks{
+			Self: selfLink,
+		},
+	}
+}
+
+type getProtoboardsResponse struct {
+	Protoboards []protoboardResponse `json:"protoboards"`
+}
+
+// Protoboards retrieves all protoboards from store
+func (s *Service) Protoboards(w http.ResponseWriter, r *http.Request) {
```

Comment:
My understanding is that we don't want to send over all of the `protoboards` data when fetching a list. Is this still the case?

If so, we should consider allowing the user to specify `view` query parameter when then retrieve all dashboards.

Reply:
in the interest of time, a separate story was created for the `view` query param i believe. cc @russorat @ebb-tide

Reply:
that's correct.